HHS/Head-Start-TTADP
# [TTAHUB-2100] Enable SES in non-production

## Description of change
This PR allows to enable AWS SES in the non-production environments by eliminating bounces to fake email addresses. It flags the generated emails with a "no-send_" prefix and filters out these addresses. It also prevents sending to duplicate emails.


## How to test
Verify on locally (while having the latest transformed db snapshot) or on Dev that notifications work as expected, e.g. by adding yourself as a collaborator.

Extra check: Verify there is no record of sending an email to a fake address by looking at the MailerLogs table.
e.g.

`SELECT * FROM "MailerLogs" order by "createdAt" desc LIMIT 10;`

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2100


## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [n/a] UI review complete

### Production Deploy


### After merge/deploy
